MinimedKit/PumpEvents/ChangeTimeFormatPumpEvent.swift

```diff
@@ -23,11 +24,18 @@ public struct ChangeTimeFormatPumpEvent: TimestampedPumpEvent {
         rawData = availableData[0..<length]
         
         timestamp = NSDateComponents(pumpEventData: availableData, offset: 2)
+        
+        func d(idx:Int) -> Int {
```

Comment:
I should probably do something about this duplicated method.  I know it's already duplicated elsewhere.